src/averageAges.js

```diff
@@ -15,11 +17,26 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let avarageAge = 0;
+  let totalAge = 0;
+  let menToCount;
+  const men = people.filter(person => person.sex === 'm');
+
+  century
+    ? menToCount = men.filter(person => (
+      Math.ceil(person.died / 100) === century
+    ))
+    : menToCount = men;
+
+  totalAge = menToCount.reduce((sum, person) => {
+    const personAge = person.died - person.born;
+
+    return sum + personAge;
+  }, 0);
+
+  avarageAge = totalAge / menToCount.length;
```

Comment:
Its not necessary to create variable in the beginning of the function only for assigning and returning.
you can just 
`return totalAge / menToCount.lengh`